contract_show_invoice/models/account_analytic_account.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2018 Alberto Martín Cortada - Guadaltech <alberto.martin@guadaltech.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = 'account.analytic.account'
+
+    @api.multi
+    def _compute_total_invoiced(self):
+        for analytic in self:
+            invoices = self.env['account.invoice'].search(
```

Comment:
Please use `read_group` technique instead for not suffering too much performance impact. See a similar example (more sophisticated) in:

https://github.com/OCA/contract/blob/5d5387e0eda83163aec2d874bfb84cde4a651e70/contract/models/res_partner.py#L19-L39